src/filefinder.h

```diff
@@ -29,9 +29,9 @@
 
 #ifdef PSP2
 #  include <psp2/types.h>
-#  define Offset SceOff
+#  define EasyRPG_Offset SceOff
 #else
-#  define Offset off_t
+#  define EasyRPG_Offset off_t
```

Comment:
This is very ugly (also the other type bending in filefinder for PSVita). I hope we can change this to use the same types later and get rid of all these `Sce` stuff. The vitasdk should really abstract this away in their c library.

Reply:
should be done in one go with the AMIGA symbol mess :/